.github/workflows/check_methoddoc_gen.yaml

```diff
@@ -0,0 +1,94 @@
+name: Check sync of functions.rst and MethodDoc.C
+
+#
+# See documentation at bottom of file
+#
+
+on:
+  pull_request:
+    paths:
+      - 'src/doc/python_scripting/functions.rst'
+      - 'src/visitpy/common/MethodDoc.C'
+      - 'src/visitpy/common/MethodDoc.h'
```

Comment:
I should delete the line for `MethodDoc.h`. It does come into play sometimes but only if `MethodDoc.C` is also in play.